src/subgraph/markets/subgraphQueries.ts

```diff
@@ -73,67 +24,18 @@ export const fetchAllMarketsDataQuery = gql`
 export const fetchMarketByIdQuery = (marketId: string) => gql`
   {
     market(id: "${marketId.toLowerCase()}") {
```

Comment:
Have we tested this? The market id in the latest snx subgraph is a number (even though IDs are stored as strings). Please check the subgraph.yaml and the returned data.
Also update the test cases

Reply:
@Akhilleshgoswami Not sure if anything was changed here. Could you please explain?

Reply:
so I have only written test case for the order, positions, and account as we were using the market from the synthetix  parifi SDK , I didn't change the code here